modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -584,9 +584,21 @@ Third party password managers can also be integrated. The only requirement is th
 password in plain text somehow. If not, additional operations might be required to get the password as
 plain text and inject it in the listener.
 
-As an example:
+Example using "Vault" as secrets store
+
+* See https://learn.hashicorp.com/collections/vault/getting-started
+on how to setup Vault as your secrets store. Then use something like the following
+
+[source,console,subs="attributes+"]
+----
+vault kv get -field=password secret/samba | {occ-command-example-prefix} wnd:listen <host> <share> <username> --password-file=-
+----
+
+Use Vault's ACLs to limit access of the token. Destroy the token after starting the service
```

Comment:
> Destroy the token after starting the service during boot with systemd.

What do you mean by that?

Reply:
Vault requires an access token, so theoretically one could provide it at boot-time to fetch the secret and then remove the token after the secret was fetched. The secret would reside in memory of the running wnd process only.  Not sure about this one either? Maybe too convoluted?

Reply:
But which token do you mean?
The wnd secret? That should stay in the vault, right?
Or the access token to the vault? But how would it be provided? Manually by a person or is this an automated step?
I probably know too little about the vault setup process... :see_no_evil:

Reply:
Ah right, I rewword